### DISCUSSION TITLE
Where/how is the text color declared in each of the widgets?

### DISCUSSION THREAD

**Discussion by hamensman** [streetturtle/awesome-wm-widgets#349] [Q&A]
Since there is not option to configure the text color, I've been trying to look in the source codes. But I'm not quite sure where in each of the widgets' source code the text color is declared.

**Comment by streetturtle** [comment]
Sorry for being late here. The color is taken by default from your theme. The idea behind not configuring color for each widget is to have consistent colors across all widget. 
If you'd like to change the color you can do it directly in the theme.lua by setting `theme.fg_normal`.
